src/Admin/ReferralAdmin.php

```diff
@@ -19,15 +19,34 @@ protected function configureListFields(ListMapper $list): void
             ->add('identifier', null, [
                 'label' => 'Numéro',
             ])
+            ->add('civility', null, [
+                'label' => 'Civilité',
+            ])
+            ->add('_fullName', null, [
+                'label' => 'Prénom/Nom',
+                'virtual_field' => true,
+                'template' => 'admin/referral/list_fullName.html.twig',
+            ])
+            ->add('_contact', null, [
+                'label' => 'Email/Téléphone',
+                'virtual_field' => true,
+                'template' => 'admin/referral/list_contact.html.twig',
+            ])
             ->add('referrer', null, [
-                'label' => 'Adhérent',
+                'label' => 'Parrain',
                 'template' => 'admin/referral/list_referrer.html.twig',
             ])
-            ->add('emailAddress', null, [
-                'label' => 'Email',
+            ->add('type', null, [
+                'label' => 'Type',
+                'template' => 'admin/referral/list_type.html.twig',
+            ])
```

Comment:
you can do the same job without custom template : 

```suggestion
            ->add('type', 'trans', [
                'label' => 'Type',
                'format' => 'referral.type.%s',
            ])
```